packages/metro-resolver/src/__tests__/package-exports-test.js

```diff
@@ -115,6 +115,7 @@ describe('with package exports resolution enabled', () => {
           '/root/node_modules/test-pkg/package.json': '',
           '/root/node_modules/test-pkg/index.js': '',
           '/root/node_modules/test-pkg/index-main.js': '',
+          '/root/node_modules/test-pkg/index-exports.js': '',
```

Comment:
The one thing with this change is we make the test case against expanding `index-exports.js` -> `index-exports.js.js` ambiguous. Instead, let's update the overriden `package.json` fields in the `'should resolve "exports" target directly'` test group.

L216:

```diff
      const context = {
        ...baseContext,
        ...createPackageAccessors({
          '/root/node_modules/test-pkg/package.json': {
-           main: 'index-main.js',
            exports: './index-exports.js',
          },
        }),
      };
```

Reply:
agreed 💯